classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
```

Comment:
[OPTIONAL] I think you might have a typo here. `@student` instead of `@students` in `initialize`